lib/medical_records/user_eligibility/configuration.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/request/camelcase'
+require 'common/client/middleware/request/multipart_request'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_custom_error'
+require 'common/client/middleware/response/mhv_errors'
+require 'common/client/middleware/response/snakecase'
+require 'faraday/multipart'
+require 'sm/middleware/response/sm_parser'
+
+module UserEligibility
+  ##
+  # HTTP client configuration for {UserEligibility::Client}
+  #
+  class Configuration < Common::Client::Configuration::REST
+    ##
+    # @return [String] Base path for dependent URLs
+    #
+    def base_path
+      "#{Settings.mhv.medical_records.host}/mhvapi/v1/usermgmt/usereligibility/"
+    end
+
+    ##
+    # @return [Hash] Headers with x-api-key and RX appToken header values for dependent URLs
+    #
+    def x_headers
+      base_request_headers.merge({
+                                   'x-api-key': Settings.mhv.medical_records.x_api_key,
+                                   appToken: Settings.mhv.rx.app_token
+                                 })
```

Comment:
Consider in the future this approach for readability

```suggestion
     mhv_headers = {
        'x-api-key': Settings.mhv.medical_records.x_api_key,
         appToken: Settings.mhv.rx.app_token
      }
      base_request_headers.merge(mhv_headers)
```

Reply:
Good call!